app.json

```diff
@@ -15,9 +19,11 @@
       "backgroundColor": "#F1FAFA"
     },
     "updates": {
-      "fallbackToCacheTimeout": 0
+      "fallbackToCacheTimeout": 4
```

Comment:
I think this is in milliseconds? Also let's do 10 seconds, and please commit only this line :p